app/controllers/pages_controller.rb

```diff
@@ -2,4 +2,6 @@ class PagesController < AuthenticatedController
   def index
     @sections = Section.all
   end
+
+  def reports; end
```

Comment:
We should remove this

Reply:
good catch, that was a remain of my first implementation.